### PR TITLE
 Task/685  implement group view page has been completed

### PR DESCRIPTION
**Pull request by jananikuruntham**

### CODE REVIEW

File: front/src/app/groups/[id]/page.tsx [hunk 1]
```diff
@@ -45,64 +329,504 @@ const GroupViewPage = () => {
...
+                              marginBottom: "0px",
+                              overflowY: "auto",
+                              border: "none",
+                              boxShadow: "none",
+                            }}
+                          >
+                            <FormGroup>
+                              {options
+                                .filter((option) =>
+                                  option.name
```
Comment (vsaravanakumar) [inline front/src/app/groups/[id]/page.tsx:722]: Handle "TypeError: Cannot read properties of null" on runtime

File: front/src/app/groups/[id]/page.tsx [hunk 2]
```diff
@@ -45,64 +329,504 @@ const GroupViewPage = () => {
...
+                                          paddingBottom: "0px",
+                                        }}
+                                      >
+                                        {option.name}
+                                      </span>
+                                    }
+                                  />
+                                ))}
+                              {options.filter((option) =>
+                                option.name
```
Comment (vsaravanakumar) [inline front/src/app/groups/[id]/page.tsx:766]: Handle "TypeError: Cannot read properties of null" on runtime

File: front/src/app/groups/[id]/page.tsx [hunk 3]
```diff
@@ -45,64 +329,504 @@ const GroupViewPage = () => {
...
+                              marginBottom: "0px",
+                              overflowY: "auto",
+                              border: "none",
+                              boxShadow: "none",
+                            }}
+                          >
+                            <FormGroup>
+                              {userOptions
+                                .filter((option) =>
+                                  option.firstName
```
Comment (vsaravanakumar) [inline front/src/app/groups/[id]/page.tsx:502]: Handle "TypeError: Cannot read properties of null" on runtime

File: front/src/app/groups/[id]/page.tsx (hunk 1 above)
Comment (jananikuruntham) [reply front/src/app/groups/[id]/page.tsx:738]: TypeError has been resolved

File: front/src/app/groups/[id]/page.tsx (hunk 2 above)
Comment (jananikuruntham) [reply front/src/app/groups/[id]/page.tsx:782]: TypeError has been resolved

File: front/src/app/groups/[id]/page.tsx (hunk 3 above)
Comment (jananikuruntham) [reply front/src/app/groups/[id]/page.tsx:509]: TypeError has been solved

**Pull request merged**